XenAdmin/Dialogs/HealthCheck/HealthCheckOverviewDialog.Designer.cs

```diff
@@ -284,6 +286,7 @@ private void InitializeComponent()
             // 
             resources.ApplyResources(this.notEnrolledPanel, "notEnrolledPanel");
             this.poolDetailsPanel.SetColumnSpan(this.notEnrolledPanel, 2);
+            this.notEnrolledPanel.Controls.Add(this.cannotAlterHealthCheckLabel, 0, 2);

```

Comment:
We need to move the label outside this panel because it does not apply only to enrollment (see other operations that are hidden when RBAC check fails). In fact notEnrolledPanel is invisible when the pool is enrolled, so the message is not seen.